views/confirm.hbs

```diff
@@ -10,6 +11,12 @@
 
     <!-- Font -->
     <link href="https://fonts.googleapis.com/css?family=Roboto+Condensed&display=swap" rel="stylesheet">
```

Comment:
You can get rid of this `<link>` since we aren't using Roboto Condensed anymore. Same thing in `error.hbs`